qtdialogs/qtdefines.py

```diff
@@ -371,15 +671,18 @@ def mouseReleaseEvent(self, ev):
          self.linkActivated.emit(ev)
 
    def enterEvent(self, ev):
-      ssStr = "QtWidgets.QLabel { background-color : %s }" % htmlColor('LBtnHoverBG')
+      ssStr = "QtWidgets.QLabel { background-color : %s }" % \
+         htmlColor('LBtnHoverBG')
       self.setStyleSheet(ssStr)
 
    def leaveEvent(self, ev):
-      ssStr = "QtWidgets.QLabel { background-color : %s }" % htmlColor('LBtnNormalBG')
+      ssStr = "QtWidgets.QLabel { background-color : %s }" % \
+         htmlColor('LBtnNormalBG')
       self.setStyleSheet(ssStr)
 
 ################################################################################
-def makeLayoutFrame(dirStr, widgetList, style=QtWidgets.QFrame.NoFrame, condenseMargins=False):
+def makeLayoutFrame(dirStr, widgetList, style=QtWidgets.QFrame.NoFrame,
```

Comment:
this kind of formatting change is fine cause it at least enforces the 80char/line rule